applications/gaming.md

```diff
@@ -0,0 +1,20 @@
+---
+description: Machine translation for gaming
+---
+
+# Gaming
+
+Machine translation for **gaming** is the translation for in-game and on-screen text, game packs, **live chat**, customer support and how-to tutorials.
+
+For gaming, machine translation is challenging because the content is creative and context-dependent. In-game machine translation is challenging because of lenght restrictions.
```

Comment:
length

We can say `In-game translation` not just `In-game machine translation`.